docker-compose.yml

```diff
@@ -26,6 +28,8 @@ services:
       - ./rezerwacje-front/package.json:/webapp/package.json
       - ./rezerwacje-front/package-lock.json:/webapp/package-lock.json
     stdin_open: true # https://github.com/facebook/create-react-app/issues/8688
+    networks:
+      - local
```

Comment:
I think it would be better to separate frontend and backend network. Here is a clear example how to do it:
https://www.callicoder.com/spring-boot-mysql-react-docker-compose-example/
Also, would be nice to add nginx.conf for the frontend serving
(see frontend Dockerfile, section: "#### Stage 2: Serve the React application" from Nginx from the link above).
We are only missing the custom nginx config in our frontend Dockerfile, so should be easy to add.
There are additional resources if needed, but I believe that first link provided will be sufficient.  
https://medium.com/yld-blog/deploy-your-create-react-app-with-docker-and-ngnix-653e94ffb537
https://stackoverflow.com/questions/54019062/how-to-send-requests-from-nginx-react-container-to-a-spring-boot-container